CRM/ManualDirectDebit/Hook/Custom/MandateDataGenerator.php

```diff
@@ -0,0 +1,264 @@
+<?php
+
+/**
+ * Generates necessary field if it wasn't filed by user
```

Comment:
rename the description to : 

```php
This class Generates the required mandate fields automatically in case they are not submitted by the user
```